addons/mail/static/src/models/messaging_initializer/messaging_initializer.js

```diff
@@ -41,6 +41,9 @@ function factory(dependencies) {
             device.start();
             const data = await this.async(() => this.env.services.rpc({
                 route: '/mail/init_messaging',
+                params: {
```

Comment:
We should maybe explain a little bit in the commit message, we add the context in the parameters of init_messaging to have the allowed_companies. 🙂